broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -12,25 +12,43 @@ const { currencies: currencyConfig } = require('../../config')
 const MINIMUM_FUNDING_AMOUNT = Big(0.00400000)
 
 /**
- * Grabs public lightning network information from relayer and opens a channel
+ * Grabs the public lightning network information from relayer and opens a channel.
  *
  * @param {Object} request - request object
  * @param {Object} request.params
  * @param {RelayerClient} request.relayer
- * @param {Logger} request.logger
  * @param {Engine} request.engines
+ * @param {Map<String, Orderbook>} request.orderbooks
+ * @param {Logger} request.logger
  * @param {Object} responses
  * @param {function} responses.EmptyResponse
  * @return {responses.EmptyResponse}
  */
-async function commit ({ params, relayer, logger, engines, orderbooks }, { EmptyResponse }) {
+async function commit ({ params, relayer, engines, orderbooks, logger }, { EmptyResponse }) {
   const { balance: balanceInCommonUnits, symbol, market } = params
   const currentCurrencyConfig = currencyConfig.find(({ symbol: configSymbol }) => configSymbol === symbol)
 
   if (!currentCurrencyConfig) {
     throw new Error(`Currency was not found when trying to commit to market: ${symbol}`)
   }
 
+  // A very archaic fee estimation amount. This number was chosen based on fees and
+  // pricing of bitcoin at the current time (Dec 7 pricing $3.5k).
+  //
+  // As a conservative number, we would typically expect fees to be between $0.10 and $0.40
+  // per on-chain transaction, so if we buffer our commitment amount by 10000sat ($0.40 usd)
+  // then we should be relatively safe.
+  //
+  // The equivalent default for litecoin would be around 1000 litoshis
```

Comment:
I think these litecoin fees are low

Reply:
@treygriffith definitely low. I've updated the amounts to `$0.50 USD`